merkuro-calendar otherwise fails to launch with:
QQmlApplicationEngine failed to load component
qrc:/main.qml:328:19: Type MainDrawer unavailable
qrc:/MainDrawer.qml:341:9: Type CheckableCollectionNavigationView unavailable
qrc:/CheckableCollectionNavigationView.qml:11:1: Failed to extract plugin meta data from '/usr/lib/qt6/qml/org/kde/merkuro/contact/libmerkuro_contact_pluginplugin.so': '/usr/lib/qt6/qml/org/kde/merkuro/contact/libmerkuro_contact_pluginplugin.so' is not a Qt plugin (metadata not found)

--- a/src/contacts/CMakeLists.txt
+++ b/src/contacts/CMakeLists.txt
@@ -76,6 +76,7 @@ target_sources(merkuro_contact_plugin PRIVATE
 
 ecm_add_qml_module(merkuro_contact_plugin
     URI "org.kde.merkuro.contact"
+    GENERATE_PLUGIN_SOURCE
     RESOURCES
         resources/fallbackBackground.png
 )
